module/web/media/js/classic/recaptcha-v2-interactive.js

```diff
@@ -0,0 +1,90 @@
+﻿/*
```

Comment:
This file contains tabs instead of spaces and is identical to module/web/media/js/pyplex/recaptcha-v2-interactive.js
Can you put it in the parent directory and reuse it in both themes?

Reply:
Damn, I thought I removed all tabs, but missed that one :( sorry.

But the two files are actually not 100% the same.... the one in pyplex uses $ from jquery to access the DOM, the other (classic template) uses $ from mootools (as jquery is not available in the classic template). And the API to acess the elements is different (jquery uses `#` to access an element by ID, mootools does not).

It's the same for the other scripts (like base.js), which are 80% the same code in both templates, but the DOM access is different, and therefore the files are "duplicated".

The file could we rewritten to be independent of jquery and mootools.... then it could be used for both templates. But I think that's a question of where pyload is heading.

If the old/classic theme will be dropped at some point and replaced by pyplex, I think it's better to use the functionality of jquery in pyplex (instead of writing everything in plain javascript), and make the necessary adjustments for the classic template in a seperate file.

If not, we can think about writing the script in plain javascript.
But that means limiting the possibilities of the more modern pyplex for the support of the old classic template....

I don't know whats the best for pyload here, maybe that's a decision which more people should be involved.